src/supermarket/app/views/api/v1/users/show.json.jbuilder

```diff
@@ -3,6 +3,7 @@ json.name @user.name
 json.company @user.company
 json.github Array(@github_usernames)
 json.twitter @user.twitter_username
+json.email @user.email
```

Comment:
Does this limit viewing a user's email in the JSON response to only admins? I couldn't find a permissions guard for this one.